<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPasswordDialog</class>
 <widget class="QDialog" name="CPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget{background: #FFFFFF;
border-radius: 8px;}</string>
  </property>
  <widget class="QLabel" name="tip_Label">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>430</y>
     <width>381</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#666666;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>1500</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox_2{background: #FFFFFF;border:none;border-top-left-radius: 8px;border-top-right-radius:8px}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0D9DDB;
border-radius: 8px 8px 0px 0px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>14</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>21</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
</string>
     </property>
     <property name="text">
      <string>管理</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="close_Btn">
     <property name="geometry">
      <rect>
       <x>1426</x>
       <y>18</y>
       <width>44</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/close.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>700</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #E4F4FC;
border-radius: 8px;
border: 3px solid #0D9DDB;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_Btn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>586</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0D9DDB;
border-radius: 8px;
color:white;</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_Btn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>586</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0D9DDB;
border-radius: 8px;
color:white;</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>tip_Label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
